storages/multipart_round_tripper.go

```diff
@@ -69,7 +69,7 @@ func (multiPartRoundTripper *MultiPartRoundTripper) Do(request *http.Request) <-
 	if backendSelectError != nil {
 		log.Debugf("Multi upload failed for %s - %s", backendSelectError, request.URL.Path)
 		go func() {
-			backendResponseChannel <- BackendResponse{Response: nil, Error: errors.New("Can't handle multi upload")}
+			backendResponseChannel <- BackendResponse{Request: request, Response: nil, Error: errors.New("Can't handle multi upload")}
```

Comment:
Don't we want to distinguish situations when can't simply handle a multipart upload and when the multipart upload failed? (we return the same error now - `errors.New("Can't handle multi upload")`)